cocotb/regression.py

```diff
@@ -241,7 +241,10 @@ def handle_result(self, result):
         """
         real_time   = time.time() - self._running_test.start_time
         sim_time_ns = get_sim_time('ns') - self._running_test.start_sim_time
-        ratio_time  = sim_time_ns / real_time
+        try:
+            ratio_time  = sim_time_ns / real_time
+        except ZeroDivisionError:
+            ratio_time = float('inf')
```

Comment:
If `sim_time_ns == 0` too, `float('nan')` would be more truthful here

Reply:
Added a check if the reported sim_time will be 0.00ns and set ratio to `float('nan')`.